### DISCUSSION TITLE
Would you share the AutoGluon Tabular codes using in Machine Hack competition

### DISCUSSION THREAD

**Discussion by sackoh** [autogluon/autogluon#1298] [General]
Recently, I read the paper "Multimodal AutoML on Structured Tables with Text Fields" @ICML2021.

and found that AG Tabular achieved 1st and 2nd places in several machine learning competitions.

If you don't mind would you share the codes used at the competitions?

I want to refer them and to get deeper understanding with using AG Tabular.

Thanks in advance!

**Comment by Innixma** [comment]
Sure! @sxjscience could you provide the code sample?

**Comment by sxjscience** [comment]
@sackoh You may check the example here: https://github.com/awslabs/autogluon/tree/master/examples/text_prediction#product-sentiment-classification

**Reply by sackoh** [reply]
Thank you!
